src/main/java/uk/gov/hmcts/reform/pcqbackend/model/PcqAnswerResponse.java

```diff
@@ -18,7 +18,7 @@
     @JsonProperty("pcq_id")
     private String pcqId;
 
-    @JsonProperty("case_id")
+    @JsonProperty("ccdCaseId")
```

Comment:
Why is this inconsistent with other json names I.e. should it be 'ccd_case_id' and not 'ccdCaseId'?

Reply:
Changed to make all properties without the underscores. This is consistent with everything else.